docs/BreakingChanges/044 HttpUtility.JavaScriptStringEncode escapes ampersand.md

```diff
@@ -0,0 +1,23 @@
+## 44: HttpUtility.JavaScriptStringEncode escapes ampersand
```

Comment:
{
  "Id": "44",
  "Title": "HttpUtility.JavaScriptStringEncode escapes ampersand",
  "Details": "Starting with the .NET Framework 4.5, JavaScriptStringEncode escapes the ampersand (&amp;) character.",
  "Suggestion": "If your app depends on the previous behavior of this method, you can add an aspnet:JavaScriptDoNotEncodeAmpersand setting to the [ASP.NET appSettings element](https://msdn.microsoft.com/en-us/library/hh975440(v=vs.110).aspx) in your configuration file.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Web.HttpUtility.JavaScriptStringEncode(System.String)",
    "M:System.Web.HttpUtility.JavaScriptStringEncode(System.String,System.Boolean)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Web.HttpUtility.JavaScriptStringEncode(System.String)
- [x] M:System.Web.HttpUtility.JavaScriptStringEncode(System.String,System.Boolean)